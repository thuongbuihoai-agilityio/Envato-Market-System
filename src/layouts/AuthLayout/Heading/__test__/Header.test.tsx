import { render } from '@testing-library/react';

// Components
import HeadingComponent from '@layouts/AuthLayout/Heading';

describe('Header', () => {
  it('should render correctly', () => {
    const { container } = render(<HeadingComponent title="Test Content" />);
    expect(container).toMatchSnapshot();
  });
});
