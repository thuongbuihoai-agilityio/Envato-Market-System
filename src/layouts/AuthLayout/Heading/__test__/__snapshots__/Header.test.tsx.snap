// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render correctly 1`] = `
<div>
  <header
    class="chakra-stack css-tl3ftk"
  >
    <h1
      class="chakra-heading css-owo776"
    >
      Test Content
    </h1>
    <p
      class="chakra-text css-1xbgeyr"
    >
      Send, spend and save smarter
    </p>
  </header>
</div>
`;
